MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+        <map:MapShapeLayer ShapesSource="https://cdn.syncfusion.com/maps/map-data/world-map.json">
+            <map:MapShapeLayer.Sublayers>
+                <map:MapLineLayer>
+                    <map:MapLineLayer.Lines>
+                        <map:MapLine StrokeThickness="2"
+                                     Stroke="#8a8a8a">
+                            <map:MapLine.From>
+                                <map:MapLatLng Latitude="77.1025"
+                                               Longitude="28.7041" />
+                            </map:MapLine.From>
+                            <map:MapLine.To>
+                                <map:MapLatLng Latitude="-106.3468"
+                                               Longitude="56.1304" />
+                            </map:MapLine.To>
+                        </map:MapLine>
+                    </map:MapLineLayer.Lines>
+                </map:MapLineLayer>
+            </map:MapShapeLayer.Sublayers>
+        </map:MapShapeLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+ public MainPage()
+{
+    InitializeComponent();
+    SfMaps maps = new SfMaps();
+    MapShapeLayer layer = new MapShapeLayer();
+    layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+    MapLineLayer mapLineLayer = new MapLineLayer();
+    MapLine line1 = new MapLine();
+    line1.From = new MapLatLng(77.1025, 28.7041);
+    line1.To = new MapLatLng(-106.3468, 56.1304);
+    line1.Stroke = Color.FromRgb(61, 155, 242);
+    line1.StrokeThickness = 2;
+    line1.StrokeLineCap = LineCap.Round;
+    layer.Sublayers.Add(mapLineLayer);
+    maps.Layer = layer;
+    this.Content = maps;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+### Arc Layer
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'MapArc'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'From'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'To'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Stroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeDashArray'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'HeightFactor'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'ControlPointFactor'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```

Comment:
The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

Reply:
Modified